### DISCUSSION TITLE
Save configuration of "Aliases" tab

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1398] [Feature Requests]
On the "Aliases" tab, I like to organize my aliases as follows:
1. Sort them alphabetically -- "Aliases A-Z"
2. View only the "Pinned Aliases"

Is there a way to save the configuration, so I don't have to re-organize my aliases each time I open the "Aliases" tab?  If not, please create a new feature to save the configuration.

Thank you.

**Comment by 4b15ed24f576** [comment]
Hey, indeed there is no way at the moment to save a filter configuration on the web. Your point feature request is valid, and we will keep that in mind for our next sprint.

In the meantime, I found a workaround that might be of use.

**Workaround for saving filters:**

Set up your filters (A-Z + Pinned Aliases) and notice that the page is refreshing and the URL changes. This new URL contains parameters that contain the filtering rules.

Save this URL and add it as a bookmark, so anytime you click on that specific URL, it will open your SimpleLogin dashboard and apply the filters previously set.

**Comment by ghost** [comment]
Very good.  Saving a shortcut to the URL will work in the interim.  I look forward to the new feature.  Thank you so much.




------- Original Message -------
